modules/dxp/apps/analytics/analytics-reports-web/test/js/components/Detail.js

```diff
@@ -253,6 +253,70 @@ describe('Detail', () => {
 		cleanup();
 	});
 
+	it('display 10 urls with view more button', async () => {
+		const testProps = {
```

Comment:
display > displays

with view more button > when clicking on the view more button?

not 100% sure about the second one, but I know @wincent will help us get it right

Reply:
Ok, I'll change it 👌🏻